src/extension/app/components/plugin/plugin-action-bar.js

```diff
@@ -58,8 +55,25 @@ export class PluginActionBar extends ConnectedElement {
     this.ready = true;
 
     reaction(
-      () => this.appStore.status,
-      () => {
+      () => this.appStore.state,
+      async () => {
+        const actionBar = await this.actionBar;
+        if (this.appStore.state === SIDEKICK_STATE.TOAST) {
+          actionBar.classList.add(this.appStore.toast.variant);
```

Comment:
Can `actionBar` be undefined?

Reply:
Yeah, I think it could be. I wrapped in an if in case. Good catch.